ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/linearoptimisation/algorithms/fillers/DiscretePstTapFiller.java

```diff
@@ -195,4 +211,19 @@ private Pair<Integer, Integer> getMinAndMaxAdmissibleTaps(PstRangeAction pstRang
         }
         return Pair.of(minAdmissibleTap, maxAdmissibleTap);
     }
+
+    private Pair<Integer, Integer> getMinAndMaxRelativeTaps(PstRangeAction pstRangeAction) {
+        int minRelativeTap = (int) -LinearProblem.infinity();
+        int maxRelativeTap = (int) LinearProblem.infinity();
+        List<TapRange> ranges = pstRangeAction.getRanges();
+        for (TapRange range : ranges) {
+            if (range.getRangeType().equals(RangeType.RELATIVE_TO_PREVIOUS_INSTANT)) {
+                minRelativeTap = Math.max(minRelativeTap, range.getMinTap());
+                maxRelativeTap = Math.min(maxRelativeTap, range.getMaxTap());
+            }
+        }
+        minRelativeTap = Math.min(0, minRelativeTap);
+        maxRelativeTap = Math.max(0, maxRelativeTap);
```

Comment:
why the min / max with 0 ? I don't think we had this before

Reply:
I'm not sure if we had it but it was to ensure we don't have a minRelativeTap which is positive. It's possible to register such a value inside the JSON crac and this would force the PST to move from one instant to another which doesn't make any sense.

Reply:
ok